packages/menu/src/test/mwc-menu.test.ts

```diff
@@ -0,0 +1,308 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// taze: chai from //third_party/javascript/chai:closure_chai
+// taze: sinon from //third_party/javascript/sinon:sinon_lib
+
+import {List} from '@material/mwc-list';
+import {ListItem} from '@material/mwc-list/mwc-list-item';
+import {Menu} from '@material/mwc-menu';
+import {MenuSurface} from '@material/mwc-menu/mwc-menu-surface';
+import {fake} from 'sinon';
```

Comment:
I eventually want to remove sinon as a dep because it is too large and thus causes IE tests to take too long. If possible can you please remove your usage of fakes. If not, it's fine can you please just file a bug to remove sinon from tests

Reply:
happy to remove it, but what would you suggest in future for stubbing things etc? our own small utils maybe?

Reply:
i added a `Fake` for now, some simple class in the util file if you wanna have a look over it